packages/db/src/models/base.ts

```diff
@@ -170,15 +170,18 @@ class BasePgModel<ItemType, RecordType extends BaseRecord> {
    *
    * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
    * @param {ItemType} item - A record to insert into the DB
-   * @returns {Promise<number[]>} List of IDs of the inserted records
+   * @param {string | Array<Ssring>} returningFields - A string or array of strings
+   *   of columns to return. Defaults to 'cumulus_id'.
+   * @returns {Promise<number[] | Object[]>} List of IDs of the inserted records
    */
   async create(
     knexOrTransaction: Knex | Knex.Transaction,
-    item: ItemType
-  ): Promise<number[]> {
+    item: ItemType,
+    returningFields: string | string[] = 'cumulus_id'
+  ): Promise<number[] | Object[]> {
```

Comment:
So looking at the [Knex docs for `returning`](https://knexjs.org/#Builder-returning), it says:

>When passed in a string, makes the SQL result be reported as an array of values from the specified column. When passed in an array of strings, makes the SQL result be reported as an array of objects, each containing a single property for each of the specified columns.

So actually we really don't have a guarantee about what the value would be if only a single column is passed - it would depend on what column is passed in. 

So I would make the return type here and in the JSdoc `Promise<unknown[] | Object[]>`

Reply:
[`bdd6661` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/bdd66615c308abb672a3d4ca00c6eb92cb5eef48#diff-3827859fa5de2f6a67108f80b2a4ec150451909cb0ef78e58ec517789311da7fR175) Thoughts on the return description?

Reply:
One minor suggestion otherwise it looks good